include/seqan3/io/record.hpp

```diff
@@ -268,6 +272,436 @@ struct record : detail::transfer_template_args_onto_t<field_types, std::tuple>
+    {
+        return get_impl(field_constant<seqan3::field::mate>{}, static_cast<base_type const &&>(*this));
+    }
+    //!\copydoc seqan3::record::mate
+    decltype(auto) mate() &
+    {
+        return get_impl(field_constant<seqan3::field::mate>{}, static_cast<base_type &>(*this));
+    }
+    //!\copydoc seqan3::record::mate
+    decltype(auto) mate() const &
+    {
+        return get_impl(field_constant<seqan3::field::mate>{}, static_cast<base_type const &>(*this));
+    }
+
+    //!\brief The mate pair information given as a std::tuple of reference name, offset and template length.
+    decltype(auto) mapping_quality() &&
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<base_type &&>(*this));
+    }
+    //!\copydoc seqan3::record::mapping_quality
+    decltype(auto) mapping_quality() const &&
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<base_type const &&>(*this));
+    }
+    //!\copydoc seqan3::record::mapping_quality
+    decltype(auto) mapping_quality() &
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<base_type &>(*this));
+    }
+    //!\copydoc seqan3::record::mapping_quality
+    decltype(auto) mapping_quality() const &
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<base_type const &>(*this));
+    }
+
+    //!\brief The cigar vector (std::vector<seqan3::cigar>) representing the alignment in SAM/BAM format.
+    decltype(auto) cigar() &&
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<base_type &&>(*this));
+    }
+    //!\copydoc seqan3::record::cigar
+    decltype(auto) cigar() const &&
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<base_type const &&>(*this));
+    }
+    //!\copydoc seqan3::record::cigar
+    decltype(auto) cigar() &
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<base_type &>(*this));
+    }
+    //!\copydoc seqan3::record::cigar
+    decltype(auto) cigar() const &
+    {
+        return get_impl(field_constant<seqan3::field::mapq>{}, static_cast<base_type const &>(*this));
+    }
+
+    //!\brief The optional tags in the SAM format, stored in a dictionary.
+    decltype(auto) tags() &&
+    {
+        return get_impl(field_constant<seqan3::field::tags>{}, static_cast<base_type &&>(*this));
+    }
+    //!\copydoc seqan3::record::tags
+    decltype(auto) tags() const &&
+    {
+        return get_impl(field_constant<seqan3::field::tags>{}, static_cast<base_type const &&>(*this));
+    }
+    //!\copydoc seqan3::record::tags
+    decltype(auto) tags() &
+    {
+        return get_impl(field_constant<seqan3::field::tags>{}, static_cast<base_type &>(*this));
+    }
+    //!\copydoc seqan3::record::tags
+    decltype(auto) tags() const &
+    {
+        return get_impl(field_constant<seqan3::field::tags>{}, static_cast<base_type const &>(*this));
+    }
+
```

Comment:
What about `bit_score` and `evalue`?

They are currently unused, but will they be removed?

Reply:
I didn't add them, because we don't use those values, yet.